### ISSUE TITLE
Error loading after install

### ISSUE THREAD

**Issue by vectorsize** [boot-clj/boot#426]
I'm having a strange error when installing boot on my machine (macbook pro / osx `10.10.5` )

Java version:
```bash
java version "1.8.0_51"
Java(TM) SE Runtime Environment (build 1.8.0_51-b16)
Java HotSpot(TM) 64-Bit Server VM (build 25.51-b03, mixed mode)
```
The error appears after a fresh install either from `homebrew`'s `boot` or `github`'s `boot.sh`

* First call to `boot` via `boot` will return:
`Downloading https://github.com/boot-clj/boot/releases/download/2.5.2/boot.jar...`
* second call to `boot` via `boot` will return:

```bash
Downloading https://github.com/boot-clj/boot/releases/download/2.5.5/boot.jar...
   clojure.lang.ExceptionInfo: Could not locate boot/task/built_in__init.class or boot/task/built_in.clj on classpath. Please check that namespaces with dashes use underscores in the Clojure file name.
    data: {:file
           "/var/folders/xt/bp8zrtt9245gsdg0wd5m10s00000gn/T/boot.user5389103512016161084.clj",
           :line 1}
java.io.FileNotFoundException: Could not locate boot/task/built_in__init.class or boot/task/built_in.clj on classpath. Please check that namespaces with dashes use underscores in the Clojure file name.
                             ...
            clojure.core/load/fn                          core.clj: 5866
               clojure.core/load                          core.clj: 5865
                             ...
           clojure.core/load-one                          core.clj: 5671
        clojure.core/load-lib/fn                          core.clj: 5711
           clojure.core/load-lib                          core.clj: 5710
                             ...
              clojure.core/apply                          core.clj:  632
          clojure.core/load-libs                          core.clj: 5749
                             ...
              clojure.core/apply                          core.clj:  634
                clojure.core/use                          core.clj: 5843
                             ...
boot.user/eval55/loading--auto--  boot.user5389103512016161084.clj:    1
                boot.user/eval55  boot.user5389103512016161084.clj:    1
                             ...
              boot.main/-main/fn                          main.clj:  192
                 boot.main/-main                          main.clj:  192
                             ...
                boot.App.runBoot                          App.java:  390
                   boot.App.main                          App.java:  467
                             ...
                       Boot.main                         Boot.java:  258
```

at the third `boot` call the error is the following:
`java.io.FileNotFoundException: Could not locate boot/aether__init.class or boot/aether.clj on classpath`

and keeps of throwing the same error from there on.

**Comment by vectorsize** [comment]
Something specific to my setup is that my home directory is in a secondary drive `/Volumes/Home` that could be one of the problems perhaps.

**Comment by vectorsize** [comment]
Checksums of all the files seemed correct, also tried `BOOT_LOCAL_REPO=/Volumes/Home/m2 boot -h`
Which installs a bunch of files and yields no errors.
But then calling `boot -h` again either with `BOOT_LOCAL_REPO=/Volumes/Home/.m2 boot -h` or just `boot -h` return nothing

**Comment by vectorsize** [comment]
Another thing tried was adding `/Volumes/Home/.m2` to my $PATH

**Comment by vectorsize** [comment]
at the third `boot` call the error is the following:
`java.io.FileNotFoundException: Could not locate boot/aether__init.class or boot/aether.clj on classpath`

**Comment by vectorsize** [comment]
FYI: Tried out a guest user, operating in `/Users/Guest` that is, everything from-and-in the same volume, as opposed to from master volume and into a secondary, and everything seems to install just fine as suspected.

That means the problem is related to operating from a secondary volume – the bug still reproduces.

**Comment by dimitertodorov** [comment]
Was able to replicate this on my MacBook.
I have a separate user profile that lives on an external drive.
If I try to run boot as that user to compile, it fails.

D

**Comment by micha** [comment]
This should be fixed with version 2.6.0, please reopen if not. Thanks!
